lib/twilio/device.ts

```diff
@@ -977,7 +977,7 @@ class Device extends EventEmitter {
     }, options);
 
     const maybeUnsetPreferredUri = () => {
-      if (this._activeCall === null && this._calls.length === 0) {
+      if (this._stream && this._activeCall === null && this._calls.length === 0) {
```

Comment:
Is there a way to add a test for this?

Reply:
[force-pushed](https://github.com/salemove/twilio-voice.js/compare/6fd91d9c3c563f8948a7f8e36cdf2e73356231f9..cf8a56e2af260e6bbcc9053b8cd3f69703913a3b)